llvm/test/SYCLLowerIR/group_local_memory.ll

```diff
@@ -0,0 +1,122 @@
+; RUN: opt -S -sycllowerwglocalmemory < %s | FileCheck --check-prefixes=CHECK,CHECK-CALL %s
```

Comment:
I suggest cutting this test significantly. The only thing you care about is calls to `__sycl_allocateLocalMemory`, so the function under test can be as simple as having only one instruction. Then you can use `update_test_checks.py` for clarity, but it's up to you.

Reply:
Reduced.